blogs/resnet/README.md

```diff
@@ -1,33 +1,79 @@
     
-## Running a Recipe
+# Selecting a Recipe
+
+As described in our [blog post](https://www.mosaicml.com/blog/mosaic-resnet):
+> We actually cooked up three Mosaic ResNet recipes – which we call Mild, Medium, and Hot – to suit a range of requirements. 
+> The Mild recipe is for shorter training runs, the Medium recipe is for longer training runs, and the Hot recipe is for the very 
+> longest training runs that maximize accuracy. 
+
+<img src="https://assets.website-files.com/61fd4eb76a8d78bc0676b47d/62a188a808b39301a7c3550f_Recipe%20Final.svg" width="50%" height="50%"/>
+
+To reproduce a specific run, two pieces of information are required:
+
+1. `recipe_yaml_path`: Path to the configuration file specifying the model and training parameters unique to each recipe.
+
+1. `scale_schedule_ratio`: Factor which scales the duration of a particular run.
+
+**Note:** The `scale_schedule_ratio` is a scaling factor for `max_duration`, each recipe sets a default `max_duration = 90ep`(epochs).  Thus a run with `scale_schedule_ratio = 0.1` will run for `90 * 0.3 = 27` epochs.
 
-1. Launch a Docker container using the `mosaicml/pytorch_vision:resnet50_recipes` Docker Image on your training system.
+First, choose a recipe you would like to work with: [`Mild`, `Medium`, `Hot`].  This will determine which configuration file, `recipe_yaml_path`, you will need to specify. 
+
+Next, determine the proper `scale_schedule_ratio` to specify to reproduce the desired run by using MosaicML's [Explorer](https://explorer.mosaicml.com).  Explorer enables users to identify the most cost effective way to run training workloads across clouds and on different types of hardware backends for a variety of models and datasets.  For this tutorial, we will focus on the [Mosaic ResNet run data](https://explorer.mosaicml.com/imagenet?sortBy=costSameQuality&model=resnet50&cloud=all&hardware=all&algorithms=all&baseline=r50_optimized_p4d&recipe=mosaicml_hot&recipe=mosaicml_medium&recipe=mosaicml_mild).
+
+The table below provides the `recipe_yaml_path` for the selected recipe and a link to the corresponding Explorer page which can be used to select a specific run and obtain the corresponding value for `scale_schedule_ratio`:
+
+   | Recipe | `recipe_yaml_path` | Explorer link |
+   | --- | --- | --- |
+   | Mild | `recipes/resnet50_mild.yaml` | [Mosaic Resnet Mild](https://explorer.mosaicml.com/imagenet?sortBy=costSameQuality&model=resnet50&cloud=all&hardware=all&algorithms=all&baseline=r50_optimized_p4d&recipe=mosaicml_mild) |
+   | Medium | `recipes/renset50_medium.yaml` | [Mosaic Resnet Medium](https://explorer.mosaicml.com/imagenet?sortBy=costSameQuality&model=resnet50&cloud=all&hardware=all&algorithms=all&baseline=r50_optimized_p4d&recipe=mosaicml_medium) |
+   | Hot | `recipes/resnet50_hot.yaml` | [Mosaic Resnet Hot](https://explorer.mosaicml.com/imagenet?sortBy=costSameQuality&model=resnet50&cloud=all&hardware=all&algorithms=all&baseline=r50_optimized_p4d&recipe=mosaicml_hot) |
+
```

Comment:
Looking at each recipe in isolation compared to a single 76.6% baseline makes it seem like the Medium and Hot recipes aren't great. Suggest adding a line like:

```suggestion

You can also compare all three recipes [here](https://explorer.mosaicml.com/imagenet?compare=recipe&sortBy=timeSameQuality&model=resnet50&cloud=mosaicml&hardware=all&algorithms=all&baseline=r50_optimized_mosaicml&recipe=mosaicml_hot&recipe=mosaicml_medium&recipe=mosaicml_mild&recipe=mosaicml_baseline).

```